### PR TITLE
feat(cli-preview): add browser option to preview command

### PR DESCRIPTION
**Pull request by kuojianlu**
To fix: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10318744

@Alive-Fish @jiayi-ruan Please help review, thanks:
1. `package.json` and `package-lock.json`: upgrade `open` version to use `open.apps` for browser selection
2. `src/telemetry/cliTelemetryEvents.ts`: add `preview-browser` telemetry property

### CODE REVIEW

File: packages/cli/src/cmds/preview/preview.ts [hunk 1]
```diff
@@ -741,7 +765,25 @@ export default class Preview extends YargsCommand {
       },
     ];
     cliLogger.necessaryLog(LogLevel.Info, utils.getColorizedString(message));
-    await open(sideloadingUrl);
+    switch (browser) {
+      case constants.Browser.chrome:
+        await open(sideloadingUrl, {
```
Comment (swatDong) [inline packages/cli/src/cmds/preview/preview.ts:770]: What if the selected browser is not installed? Any `try-catch` or error handling logic?

Comment (kuojianlu) [reply packages/cli/src/cmds/preview/preview.ts]: Good catch. Added error handling for this.

**Review by swatDong — APPROVED** [review]

**Comment by kuojianlu** [comment]
@Alive-Fish @xiaolang124 The [`open` issue](https://github.com/sindresorhus/open/issues/236) in [this PR](https://github.com/OfficeDev/TeamsFx/pull/269) has already been fixed. So it is safe to upgrade `open` version to `8.2.1`.

**Review by Alive-Fish — APPROVED** [review]
look good to me

**Pull request merged**